docs/guides/validation-user-guide.md

```diff
@@ -4,4 +4,497 @@ headline: Validation User Guide
 bodyclass: docs
 layout: docs
 ---
-<p class="coming-soon">Coming soon...</p>
+
+Spine uses Protobuf for defining domain models. It also helps us to standardize creating
+[Value Objects](https://martinfowler.com/bliki/ValueObject.html). Both those usages require
+something more than just a data structure. One of the commodities required for describing domain
+specifics is data validation. Spine builds on top of Protobuf with a Validation library.
```

Comment:
I would rephrase `Spine builds on top of Protobuf with a Validation library.` with more specifics.

```
Spine extensively uses the Validation library when working with any Protobuf messages.
```